pkgs/development/python-modules/treeo/default.nix

```diff
@@ -4,41 +4,41 @@
 , jaxlib
 , lib
 , poetry-core
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "treeo";
-  version = "0.4.0";
+  version = "0.0.9";
```

Comment:
From a semver perspective this is a downgrade. Why should we downgrade instead of upgrading?

Reply:
I see the point @dotlambda makes, it looks like upstream versioning failure indeed.